src/GovUk.Education.ExploreEducationStatistics.Publisher.Tests/GovUk.Education.ExploreEducationStatistics.Publisher.Tests.csproj

```diff
@@ -20,6 +20,7 @@
     </ItemGroup>
 
     <ItemGroup>
+        <ProjectReference Include="..\GovUk.Education.ExploreEducationStatistics.Admin.Tests\GovUk.Education.ExploreEducationStatistics.Admin.Tests.csproj" />
```

Comment:
This was to access `DbUtils` in `PublishingServiceTests.cs` for the test `PublishStagedReleaseContent`.

`PublishingService#PublishStagedReleaseContent` now needs access to the Content DB to determine what releases are superseded so it can delete the relevant `PublicationCacheKey`s.